colossalai/zero/gemini/chunk/chunk.py

```diff
@@ -6,8 +6,7 @@
 import torch.distributed as dist
 from torch.distributed import ProcessGroup
 
-from colossalai.utils import get_current_device
-from colossalai.utils.device import IS_NPU_AVAILABLE
+from colossalai.accelerator import IS_NPU_AVAILABLE, get_accelerator
```

Comment:
Can we replace `IS_NPU_AVAILABLE` with checking accelerator type?

Reply:
ok